Packs/EDL/Integrations/EDL/EDL.yml

```diff
@@ -70,6 +70,20 @@ configuration:
   name: add_comment_if_empty
   required: false
   type: 8
+- additionalinfo: 'CIDRs with prefix size smaller than this value will not be included'
```

Comment:
CIDRs with a smaller network prefix bits number will not be included. For example - if the number is 8, then 0.0.0.0/2 will be excluded from the list.

Reply:
i would like to move this option definition so that it's right after 'IP Collapsing' option